common/remote/hooks/git/gitfile.cpp

```diff
@@ -546,26 +324,27 @@ class NewGitRepositoryFileIO : implements CSimpleInterfaceOf<IFileIO>
 class NewGitRepositoryFile : public GitRepositoryFile
```

Comment:
What's the purpose of having the base class now?

Reply:
none really.  I'll combine, and rename (to remove the New prefixes)